test/integration/testHTAP.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('assert');
+const async = require('async');
+const connOption = require('./connectionOptions').valid;
+const testUtil = require('./testUtil');
+
+// Only the AWS servers support the hybrid table in the GitHub action.
+if (process.env.CLOUD_PROVIDER === 'AWS') {
+  describe('Query Context Cache test', function () {
+    this.timeout(1000000);
+    let connection;
+    
+    before((done) => {
+      connection = testUtil.createConnection(connOption);
+      async.series([
+        function (callback) {
+          testUtil.connect(connection, callback);
+        }],
+      done);
+    });
+
+    after(async () => {
+      await testUtil.destroyConnectionAsync(connection);
+    });
+
+    const querySet = [
+      {
+        sqlTexts:[
+          'create or replace database db1',
+          'create or replace hybrid table t1 (a int primary key, b int)',
+          'insert into t1 values (1, 2), (2, 3), (3, 4)'
+        ],
+        QccSize:2,
+      },
+      {
+        sqlTexts:[
+          'create or replace database db2',
+          'create or replace hybrid table t2 (a int primary key, b int)',
+          'insert into t2 values (1, 2), (2, 3), (3, 4)'
+        ],
+        QccSize:3,
+      },
+      {
+        sqlTexts:[
+          'create or replace database db3',
+          'create or replace hybrid table t3 (a int primary key, b int)',
+          'insert into t3 values (1, 2), (2, 3), (3, 4)'
+        ],
+        QccSize:4,
+      },
+      {
+        sqlTexts:[
+          'select * from db1.public.t1 x, db2.public.t2 y, db3.public.t3 z where x.a = y.a and y.a = z.a;',
+          'select * from db1.public.t1 x, db2.public.t2 y where x.a = y.a;',
+          'select * from db2.public.t2 y, db3.public.t3 z where y.a = z.a;'
+        ],
+        QccSize:4,
+      },
+    ];
+
+    function createQueryTest () {
+      const testingSet = [];
+      let testingfunction;
+      for(let i = 0; i < querySet.length; i++) {
+        const {sqlTexts,QccSize} = querySet[i];
+        for(let k = 0; k < sqlTexts.length; k++){
```

Comment:
it's new file - let's reformat it with `npm run lint:fix -- pathToThisFile`

Reply:
I already ran the lint fix before I pushed it. Is there any problem with the format?

Reply:
I would expect a space after `for` but indeed linter does not catch it - we will improve it in the future
let's leave it as it is for now

Reply:
I ran again and pushed it.